common/djangoapps/student/management/commands/_create_users.py

```diff
@@ -1,18 +1,25 @@
 """ Shared behavior between create_test_users and create_random_users """
+from django.contrib.auth import get_user_model
+from django.core.validators import ValidationError
 from xmodule.modulestore.django import modulestore
 
+
 from lms.djangoapps.instructor.access import allow_access
 from openedx.core.djangoapps.user_authn.views.registration_form import AccountCreationForm
-from common.djangoapps.student.helpers import do_create_account
+from common.djangoapps.student.helpers import do_create_account, AccountValidationError
 from common.djangoapps.student.models import CourseEnrollment
 
 
+User = get_user_model()
+
+
 def create_users(
     course_key,
     user_data,
     enrollment_mode=None,
     course_staff=False,
-    activate=False
+    activate=False,
+    ignore_user_already_exists=False,
```

Comment:
Wondering if this should be `True` by default. Probably? Even if it's a change in default behavior?

Reply:
why?